[TASK] DEVX-1954-Link-how-to-and-tutorials-from-the-documentation (#55)
pubnub/react-chat-components: lib/docs/stories/introduction/usage.stories.mdx

# Getting Started

> **TIP**: React Tutorials:

> > You can use React Chat Components to [create a group chat app](https://www.pubnub.com/tutorials/group-chat-tutorial-react/) or  [create a virtual events app](https://www.pubnub.com/tutorials/virtual-events-tutorial-react/).

Reviewer: ```suggestion
> See how to use React Chat Components to create [a group chat app](https://www.pubnub.com/tutorials/group-chat-tutorial-react/) or [a virtual events app](https://www.pubnub.com/tutorials/virtual-events-tutorial-react/).
```
